File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/org/openapitools/client/model/NameChange.scala

package org.openapitools.client.model


case class NameChange (
    /* The new player name */
    _name: String,
    /* Time which the name was changed in UNIX Timestamp without milliseconds. Usually absent for the first entry. */
    _changedToAt: Option[Long]
)
object NameChange {
    def toStringBody(var_name: Object, var_changedToAt: Object) =
        s"""
        | {
        | "name":$var_name,"changedToAt":$var_changedToAt
        | }
        """.stripMargin
}

File: generated-sources/php-laravel/mojang-api/lib/com/github/asyncmc/mojang/api/php/laravel/model/NameChange.php
<?php
/**
 * NameChange
 */
namespace com.github.asyncmc.mojang.api.php.laravel.server\com.github.asyncmc.mojang.api.php.laravel.model;

/**
 * NameChange
 */
class NameChange {

    /** @var string $name The new player name*/
    private $name;

    /** @var int $changedToAt Time which the name was changed in UNIX Timestamp without milliseconds. Usually absent for the first entry.*/
    private $changedToAt;

}
